### DISCUSSION TITLE
Access Alpine.data from Alpine.store

### DISCUSSION THREAD

**Discussion by ghost** [alpinejs/alpine#2145] [1. Help]
Hi all, i am trying access Alpine.data variables from Alpine.Store , is there a way?
thanks in advance

**Comment by SimoTod** [comment]
I don't think it's possible at the minute. What's your use case, out of curiosity? Stores are already decoupled from the markup and are not bound to the global namespace (basically what Alpine.data is made for).

**Reply by ghost** [reply]
fixed it, by getting rid of Alpine.store and moving everything into Alpine.data
